build.sbt

```diff
@@ -101,7 +102,7 @@ lazy val root = (project in file("."))
   .aggregate(common, cats, test)
 
 lazy val commonSettings = Seq(
-  scalaVersion           := "2.13.6",
+  scalaVersion           := "3.0.2",
```

Comment:
I think we should try to cross compile to 2.13 and 3 at least for a while?

Reply:
Let's just not merge it yet until we are ready to migrate everything to Scala 3. I'll change the PR to draft.

The reason being that the API changes, so we should not publish for both versions of Scala with the same library version.

In Scala 3 we have more type parameters (3) for `ReactProps` and friends which the compiler infers. In Scala 2 the compiler can't infer and we have only 1 type parameter which must be explicitly set (and the other 2 can be omitted taking advantage of existentials).

Reply:
I see, this will ripple through the facades then. Maybe we should publish only when we need it

Reply:
Could we add a fake type param to the scala 2 version?